base/cvd/cuttlefish/host/libs/web/cas/cas_downloader.cpp

```diff
@@ -0,0 +1,298 @@
+//
+// Copyright (C) 2024 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "host/libs/web/cas/cas_downloader.h"
+
+#include <android-base/strings.h>
+
+#include <cstddef>
+#include <cstdio>
+#include <map>
+#include <memory>
+#include <optional>
+#include <set>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "common/libs/utils/files.h"
+#include "common/libs/utils/result.h"
+#include "common/libs/utils/json.h"
+#include "common/libs/utils/subprocess.h"
+#include "android-base/logging.h"
+#include "json/value.h"
```

Comment:
nit: these should be a separate include group, i.e.

```c++
#include <vector>

#include <android-base/logging.h>
#include <json/value.h>

#include "common/libs/utils/files.h"
```

`android-base` and `json` are external libraries, following the [Names and Order of includes part of the style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
done.